open-metadata-implementation/access-services/data-platform/data-platform-server/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/auditlog/DataPlatformAuditCode.java

```diff
@@ -74,8 +74,73 @@
             "The Data Platform Open Metadata Access Service (OMAS) is supporting all governance zones",
             "The access service has not been passed a list of governance zones in the SupportedZones property of the access services options.  " +
                     "This means it is providing access to all Assets irrespective of the zone(s) they are located in and the created Assets can be accessed from any zone",
-            "No action is required.  This is part of the normal operation of the service.")
-            ;
+            "No action is required.  This is part of the normal operation of the service."),
+    INVALID_EVENT_FORMAT("OMAS-DATA-PLATFORM-0012",
+            OMRSAuditLogRecordSeverity.ERROR,
+            "Event{0} could not be parsed",
+            "The system is unable to process the request.",
+            "Verify the event published to the topic."),
+    PUBLISH_EVENT_EXCEPTION("OMAS-DATA-PLATFORM-0013",
+            OMRSAuditLogRecordSeverity.ERROR,
+            "Event {0} could not be published: {1}",
+            "The system is unable to process the request.",
+            "Verify the topic configuration."),
+    PROCESS_EVENT_EXCEPTION("OMAS-DATA-PLATFORM-0014",
+            OMRSAuditLogRecordSeverity.ERROR,
+            "Event {0} could not be consumed. Error: {1}",
+            "The system is unable to process the request.",
+            "Verify the topic configuration."),
+    ADD_ENTITY_EXCEPTION("OMAS-DATA-PLATFORM-0015",
+            OMRSAuditLogRecordSeverity.ERROR,
+            "Entity {0} could not be added. Error: {1}",
+            "The system is unable to process the request.",
+            "Verify the topic event."),
+    ADD_RELATIONSHIP_EXCEPTION("OMAS-DATA-PLATFORM-0016",
+            OMRSAuditLogRecordSeverity.ERROR,
+            "Relationship {0} could not be added. Error: {1}",
+            "The system is unable to process the request.",
+            "Verify the topic event."),
+    GET_ENTITY_EXCEPTION("OMAS-DATA-PLATFORM-0017",
+            OMRSAuditLogRecordSeverity.ERROR,
+            "Entity matching criteria [{0}] could not be fetched. Error: {1}",
+            "The system is unable to process the request.",
+            "Verify the topic event."),
+    GET_RELATIONSHIP_EXCEPTION("OMAS-DATA-PLATFORM-0018",
+            OMRSAuditLogRecordSeverity.ERROR,
+            "Relationship {0} could not be fetched. Error: {1}",
+            "The system is unable to process the request.",
+            "Verify the topic event."),
+    BUILD_COLUMN_CONTEXT_EXCEPTION("OMAS-DATA-PLATFORM-0019",
```

Comment:
I think some of these objects are not used, for example buildColumnContext is not in data platform flow

Reply:
Good point. :)